src/disks.py

```diff
@@ -0,0 +1,37 @@
+import dbus
+
+from moulinette.utils.log import getActionLogger
```

Comment:
Has this function disappeared? This line raises an error after rabase.

Reply:
Ah yes, you can check the other file for the new syntax, as far as i remember it's just `import logging` and then `logging.getLogger(...)`